examples/Microsoft.Spark.CSharp.Examples/Sql/Streaming/README.md

```diff
@@ -0,0 +1,112 @@
+
+Windows users can run ```nc -vvv -l -p 9999``` to connect to localhost port 9999. The result should look something like this:
+
+![NetcatConnect](https://github.com/bamurtaugh/spark/blob/StreamingLog/examples/Microsoft.Spark.CSharp.Examples/Sql/Streaming/netconnect.PNG)
+
+Our Spark program will be listening for input we type into this command prompt.
+
+#### Connect to Stream: ReadStream()
+
+The `ReadStream()` method returns a `DataStreamReader` that can be used to read streaming data in as a `DataFrame`. We'll include the host and port information so that our Spark app knows where to expect its streaming data.
+
+```CSharp
+DataFrame words = spark
+      .ReadStream()
+      .Format("socket")
+      .Option("host", hostname)
+      .Option("port", port)
+      .Load();
+```
+
+### 3. Register a UDF
+
+A UDF is a *user-defined function.* We can use UDFs in Spark applications to perform calculations and analysis on our data.
+
+```CSharp
+spark.Udf().Register<string, bool>("MyUDF", input => ValidLogTest(input));
+```
+
+In the above code snippet, we register a UDF that will pass each log entry it receives to the method *ValidLogTest.*
+
+### 4. Use Spark SQL
+
+Next, we'll use Spark SQL to make SQL calls on our data. It's common to combine UDFs and Spark SQL so that we can apply a UDF to each 
+row of our DataFrame.
+
+```CSharp
+DataFrame sqlDf = spark.Sql("SELECT WordsEdit.value, MyUDF(WordsEdit.value) FROM WordsEdit"); 
+```
+
+### 5. Display Your Stream
+
+We can use `DataFrame.WriteStream()` to establish characteristics of our output, such as printing our results to the console and only displaying the most recent output and not all of our previous output as well. 
+
+```CSharp
+Spark.Sql.Streaming.StreamingQuery query = sqlDf
+      .WriteStream()
+      .Format("console")
+      .Start();
+```
+
+### 6. Running Your Code
+
+Structured streaming in Spark processes data through a series of small **batches.** 
+When you run your program, the command prompt where we established the netcat will allow you to start typing.
+In our example, when you hit *enter* after entering data in the command prompt, Spark will consider that a batch and run the UDF. 
+
+![StreamingOutput](https://github.com/bamurtaugh/spark/blob/StreamingLog/examples/Microsoft.Spark.CSharp.Examples/Sql/Streaming/streamingnc.PNG)
+
+Checkout the directions for building and running this app on [Windows](../../../../../../docs/building/windows-instructions.md) or [Ubuntu](../../../../../../docs/building/ubuntu-instructions.md).
+
+#### Windows Example:
+
+After starting a new netcat session, open a new terminal and run your `spark-submit` command, similar to the following:
+
+```CSharp
+spark-submit --class org.apache.spark.deploy.dotnet.DotnetRunner --master local C:\GitHub\spark\src\scala\microsoft-spark-2.4.x\target\microsoft-spark-2.4.x-0.6.0.jar Microsoft.Spark.CSharp.Examples.exe Sql.Streaming.StructuredNetworkWordCount localhost 9999
```

Comment:
This command tells users to run `Sql.Streaming.StructuredNetworkWordCount` but the example in this document doesn't match what `Sql.Streaming.StructuredNetworkWordCount` does.

Reply:
Changed to reference new udf sample